### PR TITLE
Youth explorer national launch

### PR DESCRIPTION
**Pull request by longhotsummer**
* [x] find better main image

### CODE REVIEW

File: _articles/youth-explorer-national.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: article
+author: Greg
```
Comment (guushoekman) [inline _articles/youth-explorer-national.md]: `author: greg`

Comment (longhotsummer) [inline _articles/youth-explorer-national.md]: :D

File: _articles/youth-explorer-national.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+To get started, visit [YouthExplorer.org.za](https://youthexplorer.org.za) and take a look at our series of [explainer videos to help you](https://youthexplorer.org.za/help) get the most out of Youth Explorer.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/do5XexPJhDc?list=PL7MJ_sFHs952rlUTNpQr7-n9UwQWkBXyx" frameborder="0" allowfullscreen=""></iframe>
+
+We also have other videos that cover:
+
+* [the table view and downloading data](https://www.youtube.com/watch?list=PL7MJ_sFHs952rlUTNpQr7-n9UwQWkBXyx&v=bynx320D5kY),
+* [exploring data on a map](https://www.youtube.com/watch?list=PL7MJ_sFHs952rlUTNpQr7-n9UwQWkBXyx&v=IHVWdv-Yuos),
+* [comparing the distrubition of data between different places](https://www.youtube.com/watch?list=PL7MJ_sFHs952rlUTNpQr7-n9UwQWkBXyx&v=jhzGFGSAJ9o), and
```
Comment (xybrnet) [inline _articles/youth-explorer-national.md]: * distribution

Comment (longhotsummer) [inline _articles/youth-explorer-national.md]: thanks

**Pull request merged**